templates/azure-health.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+# Based on https://learn.microsoft.com/en-us/azure/virtual-machines/linux/no-agent#add-required-code-to-the-vm
+# The goal of this script is to report to Azure that this VM is up and running in the right way. The IP address is hardcoded because it is the API of the Azure, https://learn.microsoft.com/en-us/azure/virtual-network/what-is-ip-address-168-63-129-16, which also states that the IP address will not change.
+
+# The original script uses curl for making HTTP requests, however, curl is not installed on our K8S-OS. Therefore the following HTTP request will have to be done in the old way of writing each line in the HTTP request manually.
+# The below loop is about fetching the goalstate of the VM along with other basic information. It will attempt five times before exiting with a failed exitcode.
+attempts=1
+until (("$attempts" > 5)); do
```

Comment:
Neither quotation nor `$` is required.
```suggestion
until ((attempts > 5)); do
```